toyapp/Gemfile

```
@@ -0,0 +1,33 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+gem 'rails',      '6.0.3.2'
+gem 'puma',       '4.3.5'
+gem 'sass-rails', '6.0.0'
+gem 'webpacker',  '4.2.2'
+gem 'turbolinks', '5.2.1'
+gem 'jbuilder',   '2.10.0'
+gem 'bootsnap',   '1.4.6', require: false
+group :development, :test do
```

```suggestion

group :development, :test do
```
add new lines before & after gem groups for clarity